app/views/layouts/application.html.haml

```diff
@@ -18,8 +18,7 @@
         = ' - '
       = title
 
-    = stylesheet_pack_tag 'common', media: 'all'
-    = stylesheet_pack_tag stylesheet_for_layout, media: 'all'
```

Comment:
OK, it took me a bit to understand, but I see now that there is no `public.css` vs` application.css` – it's all the same CSS. So it really is pointless to have a `common.css` because all entry points require exactly the same CSS.